include/micm/process/arrhenius_rate_constant.hpp

```diff
@@ -55,14 +55,12 @@ namespace micm
   };
 
   inline ArrheniusRateConstant::ArrheniusRateConstant()
-      : parameters_()
-  {
-  }
+    : parameters_()
+    {}
```

Comment:
I typically use clang-format to auto format all of our code. Did you happen to run that?

Reply:
No I didn't. I wanted to ask about formatting. Good to know. I will use the same auto format and update the code.

Reply:
@K20shores do you have specific config for clang-format or use the default?

Reply:
There's a .clang-format file in the root of the repository. Feel free to make any additions to it that you like. Again, we don't really care about which format we have, so long as we have a format.